### ISSUE TITLE
Error:Error: error in parsing "g/"

### ISSUE THREAD

**Issue by jp1017** [Nightonke/CoCoin#4]
as编译最新项目报错：

![qq 20160117215955](https://cloud.githubusercontent.com/assets/7868514/12377833/b0607416-bd65-11e5-8c24-e20969ddf619.png)

**Comment by Nightonke** [comment]
https://github.com/wnafee/vector-compat/issues/30#issuecomment-163012205
看看这个能不能解决，话说这个文件好像没用到的样子，如果不行你可以试试看删掉。

**Comment by jp1017** [comment]
还是不行，并且编译超级慢，as2.0preview5

**Comment by NaughtBoy** [comment]
请问你的图片是怎么上传的？是先存在自己的仓库里面再把链接拷出来写成markdown吗？

**Comment by Nightonke** [comment]
The pictures in README.md were firstly uploaded to github(the PNG and GIF folder) then their corresponding links were written.
This application is a complete product which you can search it on Baidu market(other markets later).

**Comment by NaughtBoy** [comment]
谢谢，但是我想问的是issues里面的图片是怎么加上去的。
ps：附上一个bug，在启动页，当动画快结束的时候退出程序再重新进去，MainActivity会多次启动。

**Comment by Nightonke** [comment]
Same as insert a picture in README.md.
Write this ![shown name](picture link) in issue and click preview you can see the picture.
This bug is easy to fix and I will do it later. Just override the OnBackPressed method in the splash activity.

**Comment by NaughtBoy** [comment]
OK. thanks for your help!

**Comment by NaughtBoy** [comment]
另外，还有个UI遮挡的问题，我录入了一个￥22222 的条目，然后在【账单-这周】 下面的柱形图左侧标注最大是五位数，超出了布局的范围，万位的数字只有一半，这个也是适配的问题吧？然后在同个页面上下滚动的时候，有时候下面的布局会顶到toolbar里面，卡在缩小的CoCoin和TabHost的中间。这个不是必现的。

**Comment by Nightonke** [comment]
Yes, the first bug is just a margin or padding bug. Just set the paddingLeft a little bigger to give more space to the y-axis. I will fix it later.
The second one is a bug of a open-source library. I will check this.

**Comment by jp1017** [comment]
@NaughtBoy issue里上传图片了，直接把图片拖到这个输入框里就可以的，so easy.

**Comment by NaughtBoy** [comment]
@jp1017 O(∩_∩)O谢谢

**Comment by zhongchin** [comment]
我也遇到同样的问题,我把那个全部删掉了，能安装了，但是不能运行

**Comment by zhongchin** [comment]
刚百度了一下，不用删除xml文件里面的东西，点击build->clean project然后请出去一下cache(file->invalidate caches)就ok了

**Comment by xjh1994** [comment]
把drawable-v14里的XML文件移到drawable-v21就行了，是API版本问题

**Comment by hatimmakki** [comment]
add this
    defaultConfig {
        ...
        vectorDrawables.useSupportLibrary = true
    }

**Comment by yongjiexu** [comment]
我的解决办法是：把包含“/g”的行删掉。It works fine
